src/equations/polytropic_euler_2d.jl

```diff
@@ -0,0 +1,172 @@
+    varnames(::typeof(cons2prim), ::PolytropicEulerEquations2D) = ("rho", "v1", "v2")
+
+    # Calculate 1D flux for a single point in the normal direction
+    # Note, this directional vector is not normalized
+    @inline function flux(u, normal_direction::AbstractVector,
+                          equations::PolytropicEulerEquations2D)
+        rho_e = last(u)
+        rho, v1, v2 = cons2prim(u, equations)
+        p = pressure(u, equations)
+
+        v_normal = v1 * normal_direction[1] + v2 * normal_direction[2]
+        rho_v_normal = rho * v_normal
+        f1 = rho_v_normal
+        f2 = rho_v_normal * v1 + p * normal_direction[1]
+        f3 = rho_v_normal * v2 + p * normal_direction[2]
+        return SVector(f1, f2, f3)
+    end
+
+    @inline function flux_ranocha(u_ll, u_rr, normal_direction::AbstractVector,
+                                  equations::PolytropicEulerEquations2D)
+        # Unpack left and right state
+        rho_ll, v1_ll, v2_ll = cons2prim(u_ll, equations)
+        rho_rr, v1_rr, v2_rr = cons2prim(u_rr, equations)
+        p_ll = equations.kappa * rho_ll^equations.gamma
+        p_rr = equations.kappa * rho_rr^equations.gamma
+        v_dot_n_ll = v1_ll * normal_direction[1] + v2_ll * normal_direction[2]
+        v_dot_n_rr = v1_rr * normal_direction[1] + v2_rr * normal_direction[2]
+
+        # Compute the necessary mean values
+        rho_mean = ln_mean(rho_ll, rho_rr)
+        # Algebraically equivalent to `inv_ln_mean(rho_ll / p_ll, rho_rr / p_rr)`
+        # in exact arithmetic since
+        #     log((ϱₗ/pₗ) / (ϱᵣ/pᵣ)) / (ϱₗ/pₗ - ϱᵣ/pᵣ)
+        #   = pₗ pᵣ log((ϱₗ pᵣ) / (ϱᵣ pₗ)) / (ϱₗ pᵣ - ϱᵣ pₗ)
+        inv_rho_p_mean = p_ll * p_rr * inv_ln_mean(rho_ll * p_rr, rho_rr * p_ll)
+        v1_avg = 0.5 * (v1_ll + v1_rr)
+        v2_avg = 0.5 * (v2_ll + v2_rr)
+        p_avg = 0.5 * (p_ll + p_rr)
+
+        # Calculate fluxes depending on normal_direction
+        f1 = rho_mean * 0.5 * (v_dot_n_ll + v_dot_n_rr)
+        f2 = f1 * v1_avg + p_avg * normal_direction[1]
+        f3 = f1 * v2_avg + p_avg * normal_direction[2]
+
+        return SVector(f1, f2, f3)
+    end
+
+    @inline function min_max_speed_naive(u_ll, u_rr, normal_direction::AbstractVector,
+                                         equations::PolytropicEulerEquations2D)
+        rho_ll, v1_ll, v2_ll = cons2prim(u_ll, equations)
+        rho_rr, v1_rr, v2_rr = cons2prim(u_rr, equations)
+        p_ll = equations.kappa * rho_ll^equations.gamma
+        p_rr = equations.kappa * rho_rr^equations.gamma
+
+        v_normal_ll = v1_ll * normal_direction[1] + v2_ll * normal_direction[2]
+        v_normal_rr = v1_rr * normal_direction[1] + v2_rr * normal_direction[2]
+
+        norm_ = norm(normal_direction)
+        # The v_normals are already scaled by the norm
+        λ_min = v_normal_ll - sqrt(equations.gamma * p_ll / rho_ll) * norm_
+        λ_max = v_normal_rr + sqrt(equations.gamma * p_rr / rho_rr) * norm_
+
+        return λ_min, λ_max
```

Comment:
```suggestion
        lambda_min = v_normal_ll - sqrt(equations.gamma * p_ll / rho_ll) * norm_
        lambda_max = v_normal_rr + sqrt(equations.gamma * p_rr / rho_rr) * norm_

        return lambda_min, lambda_max
```
I think we recently had this fixed in all other equations